htdocs/compta/resultat/clientfourn.php

```diff
@@ -250,6 +250,9 @@
 
 print '<table class="noborder centpercent">';
 print '<tr class="liste_titre">';
+if ($modecompta != 'CREANCES-DETTES') {
+	print_liste_field_titre('');  // Make 4 columns in total whatever $modecompta is
```

Comment:
We can target to have same nb of columns for both "CREANCES-DETTES" and other modes, but it would be better if the fix is done on line 265 by adding a 
```
else {
print_liste_field_titre(''); // Make 4 columns in total whatever $modecompta is
}
```

So each print with a condition is counterbalanced by a print if condition has the opposite value. Would be better for code readability.